src/Microsoft.Fx.Portability/CompressedHttpClient.cs

```diff
@@ -14,22 +14,38 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Security.Authentication;
 using System.Threading.Tasks;
 
 namespace Microsoft.Fx.Portability
 {
     internal class CompressedHttpClient : HttpClient
     {
+        internal const SslProtocols SupportedSSLProtocols = SslProtocols.Tls12;
```

Comment:
Is this only used under FEATURE_CLIENT_HANDLER_SET_SSL_PROTOCOLS?

Reply:
Yes, I'll move it under an #if to make that clear.